src/pages/help-centre/_faq-schema.ts

```diff
@@ -1,6 +1,23 @@
 import { localize } from 'components/localization'
 
-export const faq_schema = {
+type FaqSchemaType = {
+    '@context': string
+    '@type': string
+    mainEntity: {
+        '@type': string
+        name: string
+        acceptedAnswer: {
+            '@type': string
+            text: string
+        }
+        acceptedAnswerEU?: {
+            '@type': string
+            text: string
+        }
+    }[]
+}
```

Comment:
You can create a `mainEntityType` to make this cleaner

```suggestion
type AcceptedAnswerType = {
       '@type': string
       text: string
}

type MainEntityType = {
        '@type': string
        name: string
        acceptedAnswer: AcceptedAnswerType
        acceptedAnswerEU?: AcceptedAnswerType
   }

type FaqSchemaType = {
    '@context': string
    '@type': string
    mainEntity: MainEntityType[]
}
```